odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -14,8 +14,10 @@
         "[13] Use of deprecated decorator '@api.returns'",
         r".*@api.cr.*\n":
         "[13] Use of deprecated decorator '@api.cr'",
-        r".*@api.model_cr.*\n":
+        r".*@api.model_cr\s*\n":
```

Comment:
I think *word boundary* fits better.

```suggestion
        r"\b@(api\.)?model_cr\b*":
```

Same below. Just a suggestion.